pkg/apis/controller/experiments/v1beta1/experiment_types.go

```diff
@@ -207,12 +207,23 @@ const (
 )
 
 type FeasibleSpace struct {
-	Max  string   `json:"max,omitempty"`
-	Min  string   `json:"min,omitempty"`
-	List []string `json:"list,omitempty"`
-	Step string   `json:"step,omitempty"`
+	Max          string       `json:"max,omitempty"`
+	Min          string       `json:"min,omitempty"`
+	List         []string     `json:"list,omitempty"`
+	Step         string       `json:"step,omitempty"`
+	Distribution Distribution `json:"distribution,omitempty"`
 }
 
+type Distribution string
+
+const (
+	DistributionUniform     Distribution = "uniform"
+	DistributionLogUniform  Distribution = "logUniform"
+	DistributionNormal      Distribution = "normal"
+	DistributionLogNormal   Distribution = "logNormal"
+	DistributionCategorical Distribution = "categorical"
```

Comment:
We should remove this.

Reply:
Yes, removed Categorical from `Distribution` type.